### PR TITLE
Updating meeting info for SIG-Windows

### PR DESCRIPTION
**Pull request by marosset**
Updating meetings for SIG-Windows

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by marosset** [comment]
@jsturtevant @aravindhp @immuzz  FYI

File: sig-list.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ When the need arises, a [new SIG can be created](sig-wg-lifecycle.md)
...
-|[Windows](sig-windows/README.md)|windows|* [Mark Rossetti](https://github.com/marosset), Microsoft<br>|* [Slack](https://kubernetes.slack.com/messages/sig-windows)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-windows)|* Regular SIG Meeting: [Tuesdays at 12:30 Eastern Time (ET) (weekly)](https://zoom.us/j/96892680257?pwd=TVNyMzB4VVMwRGZnUkgzT1dnb2szZz09)<br>
+|[Windows](sig-windows/README.md)|windows|* [Mark Rossetti](https://github.com/marosset), Microsoft<br>|* [Slack](https://kubernetes.slack.com/messages/sig-windows)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-windows)|* Backlog Grooming / Bug Triage Meeting: [Thursdays at 12:30 Eastern Time (ET) (biweekly)](https://zoom.us/j/94389601840?pwd=MCs2SEJQWG0zUWpBS3Nod0ZNMmVXQT09)<br>* Regular SIG Meeting: [Tuesdays at 12:30 Eastern Time (ET) (weekly)](https://zoom.us/j/96892680257?pwd=TVNyMzB4VVMwRGZnUkgzT1dnb2szZz09)<br>* Weekly CI Meeting: [Tuesdays at 12:15 Eastern Time (ET) (weekly)](https://zoom.us/j/96892680257?pwd=TVNyMzB4VVMwRGZnUkgzT1dnb2szZz09)<br>
```
Comment (aravindhp) [inline sig-list.md:58]: I suggest we use `Backlog Refinement`, given `Grooming` has bad connotations.

File: sigs.yaml [hunk 2]
```diff
@@ -2530,6 +2530,12 @@ sigs:
     - github: michmike
       name: Michael Michael
   meetings:
+  - description: Backlog Grooming / Bug Triage Meeting
```
Comment (aravindhp) [inline sigs.yaml:2533]: Same suggestion here.

File: sig-windows/README.md [hunk 3]
```diff
@@ -13,9 +13,12 @@ Focuses on supporting Windows Node and scheduling Windows containers on Kubernet
 The [charter](charter.md) defines the scope and governance of the Windows Special Interest Group.
 
 ## Meetings
+* Backlog Grooming / Bug Triage Meeting: [Thursdays at 12:30 Eastern Time (ET)](https://zoom.us/j/94389601840?pwd=MCs2SEJQWG0zUWpBS3Nod0ZNMmVXQT09) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=12:30&tz=Eastern%20Time%20%28ET%29).
```
Comment (aravindhp) [inline sig-windows/README.md:16]: Same suggestion here

**Review by aravindhp — COMMENTED** [review]
Thanks for doing this, @marosset

File: sigs.yaml (hunk 2 above)
Comment (marosset) [reply sigs.yaml:2533]: The other files are auto-generated for this one.
Good suggestion, let me update.

File: sig-list.md (hunk 1 above)
Comment (marosset) [reply sig-list.md]: Updating, thanks for the suggestion!

**Review by aravindhp — COMMENTED** [review]
/lgtm

**Pull request merged**